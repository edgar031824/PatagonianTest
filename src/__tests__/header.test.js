import renderer from 'react-test-renderer';
import React from 'react';
import Header from '../components/header';
import { cleanup, render } from '@testing-library/react';
describe('Header component', () => {
	afterEach(() => {
		cleanup();
	});

	it('should match snapshot', () => {
		const tree = renderer.create(<Header />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});

it('should render a wrapper component', () => {

	const { getByTestId } = render(<Header />);
	expect(getByTestId('wrapper')).toBeDefined();
});