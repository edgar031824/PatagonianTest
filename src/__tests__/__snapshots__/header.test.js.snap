// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<div
  className="sc-bdnylx bMssuQ"
  data-testid="wrapper"
>
  <img
    alt=""
    className="sc-gtssRu bQZeWh"
    src="https://patagonian.it/assets/img/logo-nbar.png"
  />
</div>
`;
